compositeviews/compositeview01/es_projection_query.rq

```diff
@@ -0,0 +1,23 @@
+PREFIX schema: <http://schema.org/>
+PREFIX nxv: <https://bluebrain.github.io/nexus/vocabulary/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
```

Comment:
Could you maybe add a short sparql comment about the number of levels and the filtering of the graph with the `validate` expressions? Maybe refer to the lines we discussed today in the rdf2rdf project.

Reply:
done in https://github.com/Connectome-Consortium/rescs_shacl_shapes/pull/21/commits/77fcbdf2531b6a201d46e242c60088d6e0e05037